src/librustc_passes/weak_lang_items.rs

```diff
@@ -59,7 +59,7 @@ fn verify<'tcx>(tcx: TyCtxt<'tcx>, items: &lang_items::LanguageItems) {
     }
 
     for (name, &item) in WEAK_ITEMS_REFS.iter() {
-        if missing.contains(&item) && !whitelisted(tcx, item) && items.require(item).is_err() {
+        if missing.contains(&item) && !not_required(tcx, item) && items.require(item).is_err() {
```

Comment:
"if missing contains (item), and not whitelisted (item)" vs "if missing contains (item), and not not required (item)"

I feel the latter is MUCH harder to interpret than the former. Is "not not required" equivalent to "required" here? If so, could the return value be inverted and name changed usage is more clear?

(even though I see absolutely no problem with the original)

Reply:
Done.